website/docs/reference/actions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Actions
+---
+
+:::info Availability
+
+Unleash actions is an **enterprise-only** feature released as beta in Unleash v5.11.
+
+:::
+## Overview
+
+Actions enable our users to execute automated actions based on signals received from an external system.
+
+![Actions executed by Unleash.](/img/signals/action-execution.png)
+
+## Creating actions
+
+:::info Permissions
+
+During the beta phase, creating actions require admin permissions.
```

Comment:
I also assume this will be a project-level permission in the future, but I think for now it's safer to just mention this.
```suggestion
Creating actions requires the `ADMIN` permission.
```